resources/lang/fr.js

```diff
@@ -83,20 +83,57 @@ class CactbotLanguageFr extends CactbotLanguage {
       Pull: 'Pull',
     });
 
+    this.kPetNames = Object.freeze([
+      // Pulled from https://xivapi.com/Pet?pretty=true&language=fr
+      'Carbuncle émeraude',
+      'Carbuncle topaze',
+      'Ifrit-Egi',
+      'Titan-Egi',
+      'Garuda-Egi',
+      'Eos',
+      'Selene',
+      'Auto-tourelle Tour',
+      'Auto-tourelle Fou',
+      'Demi-Bahamut',
+      'Demi-Phénix',
+      'Séraphin',
+      'Carbuncle hécatolite',
+      'Estime',
+      'Automate Reine',
+      'Ombre',
+    ]);
+
     this.countdownStartRegex = function() {
-      return Regexes.parse(/Début du combat dans (\y{Float}) secondes !/);
+      return Regexes.gameLog({
+        capture: true,
+        line: 'Début du combat dans (?<time>\\y{Float}) secondes !',
+      });
     };
     this.countdownEngageRegex = function() {
-      return /:À l'attaque !/;
+      return Regexes.gameLog({
+        capture: true,
+        line: 'À l\'attaque !',
+      });
     };
     this.countdownCancelRegex = function() {
-      return /Le compte à rebours a été interrompu par /;
+      return Regexes.gameLog({
+        capture: true,
+        // FIXME No name capture for person that cancelled
```

Comment:
@mooondark @VanSlanzar Could one of y'all provide a log line for cancelling the countdown?

Reply:
00|2020-05-06T16:39:14.0000000+02:00|00b9||Le compte à rebours a été interrompu par vous.|71603837bb2739a570edeffa603e2b53

I wasn't in a group so I interrupted it myself (vous).

Reply:
00|2020-05-06T16:27:07.0000000+02:00|00b9||Le compte à rebours a été interrompu par Van S..|358818449db610fb2157fc45fc4c21c0
I was alone.